Formula/math-comp.rb

```diff
@@ -0,0 +1,46 @@
+class MathComp < Formula
+  desc "Mathematical Components for the Coq proof assistant"
+  homepage "https://math-comp.github.io/math-comp/"
+  url "https://github.com/math-comp/math-comp/archive/mathcomp-1.6.4.tar.gz"
+  sha256 "c672a4237f708b5f03f1feed9de37f98ef5c331819047e1f71b5762dcd92b262"
+  head "https://github.com/math-comp/math-comp.git"
+
+  option "without-docs", "Don't generate or install documentation"
+
+  depends_on "ocaml"
+  depends_on "coq"
+
+  def install
+    ENV["COQBIN"] = "#{Formula["coq"].bin}/"
```

Comment:
Should use `opt_bin` not `bin`. Also, is the trailing slash really needed?

Reply:
The trailing slash is necessary.

> COQDEP=$(COQBIN)coqdep

from Makefile.

Reply:
ok.